packages/origin-ui-core/src/components/Organization/OrganizationView.tsx

```diff
@@ -1,11 +1,228 @@
+            });
+        }
+    }, [userOffchain]);
+
+    if (!formValues) {
+        return <Skeleton variant="rect" height={200} />;
+    }
+
+    return (
+        <Paper className={classes.container}>
+            <Grid container spacing={3}>
+                <Grid item xs={6}>
+                    <TextField
+                        label="Code"
+                        className="mt-3"
+                        value={formValues.code}
+                        fullWidth
+                        disabled
+                    />
+
+                    <TextField
+                        label="Name"
+                        className="mt-3"
+                        value={formValues.name}
+                        fullWidth
+                        disabled
+                    />
+
+                    <TextField
+                        label="Headquarters Country"
+                        value={formValues.headquartersCountry}
+                        className="mt-3"
+                        fullWidth
+                        disabled
+                    />
+
+                    <TextField
+                        label="Contact"
+                        className="mt-3"
+                        value={formValues.contact}
+                        fullWidth
+                        disabled
+                    />
+
+                    <TextField
+                        label="Email"
+                        className="mt-3"
+                        value={formValues.email}
+                        fullWidth
+                        disabled
+                    />
+
+                    <TextField
+                        label="Telephone"
+                        className="mt-3"
+                        value={formValues.telephone}
+                        fullWidth
+                        disabled
+                    />
+
+                    <TextField
+                        label="Company number"
+                        value={formValues.companyNumber}
+                        disabled
+                        className="mt-3"
+                        fullWidth
+                    />
+
+                    <TextField
+                        label="VAT number"
+                        value={formValues.vatNumber}
+                        disabled
+                        className="mt-3"
+                        fullWidth
+                    />
+
+                    <TextField
+                        label="CEO name"
+                        value={formValues.ceoName}
+                        disabled
+                        className="mt-3"
+                        fullWidth
+                    />
+
+                    <TextField
+                        label="CEO passport number"
+                        value={formValues.ceoPassportNumber}
+                        disabled
+                        className="mt-3"
+                        fullWidth
+                    />
```

Comment:
Why do we keep those fields? They are not part of the new designs.

Reply:
Its just renamed OrganizationForm. I didn't changed it in this PR because PR is only about new organization registration form